digdag-docs/src/command_reference.rst

```diff
@@ -386,7 +386,9 @@ In the config file, following parameters are available
 * log-server.local.split_size (long. default: 0  (not splitted))
 * digdag.secret-encryption-key = (base64 encoded 128-bit AES encryption key)
 * executor.task_ttl (string. default: 1d. A task is killed if it is running longer than this period.)
+* executor.task_max_run (integer. default: 1000. Max number of tasks in workflow.)
 * executor.attempt_ttl (string. default: 7d. An attempt is killed if it is running longer than this period.)
+* executor.attempt_max_run (integer. default: 100. Max number of running attempts per each site_id.)
```

Comment:
How about adding term `at once` or `in concurrently` in the description?

Reply:
Thanks, will fix it.

Reply:
Done.